pkg/kubelet/kubelet.go

```diff
@@ -289,8 +289,7 @@ func PreInitRuntimeService(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 	containerRuntime string,
 	runtimeCgroups string,
 	remoteRuntimeEndpoint string,
-	remoteImageEndpoint string,
-	nonMasqueradeCIDR string) error {
```

Comment:
this function is covered in this PR https://github.com/kubernetes/kubernetes/pull/107594/files

Reply:
Not really important which one covers this first. The other one will be rebased.